modules/electromagnetics/src/auxkernels/AuxComplexHeating.C

```diff
@@ -0,0 +1,39 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "AuxComplexHeating.h"
+
+registerMooseObject("ElectromagneticsApp", AuxComplexHeating);
+
+InputParameters
+AuxComplexHeating::validParams()
+{
+  InputParameters params = AuxKernel::validParams();
+  params.addClassDescription("Computes the heating due to the electic field in the "
+                             "form of $(0.5Re(\\sigma E \\cdot E^{*} ))$");
+  params.addRequiredCoupledVar("E_real", "The real component of the E-field.");
+  params.addRequiredCoupledVar("E_imag", "The imaginary component of the E-field.");
```

Comment:
Let's not forget to switch to "electric field" from "E-field" in these too.

Reply:
Edited in [70a7473](https://github.com/idaholab/moose/pull/28759/commits/70a7473b7affe5e6fce2a562654a3f55bc306c9f)